manual/manual/tutorials/coreexamples.etex

```diff
@@ -580,6 +580,80 @@ let fixpoint f x =
 the function "f" cannot raise a "Done" exception, which removes an
 entire class of misbehaving functions.
 
+\section{Lazy expressions}
+
+OCaml allows us to defer some computation, usually expensive one, until later when 
+we need the result of that computation. 
+
+We use "lazy (expr)" to delay the evaluation of some expression "expr". For 
+example, we can defer the computation of "1+1" until we need the result of that
+expression, i.e. "2". Let us see how we initialize a lazy expression. 
```

Comment:
Maybe just ` "2"` rather than `i.e. "2"`?

Reply:
I tried to change it to simply `"2"`, but it reads "the result of that expression 2", doesn't it?

Reply:
as if our expression is 2 not 1+1

Reply:
Well, the comma would still be here:

> the result of that expression, "2".

or an em dash (`---` in latex) could work too:

> the result of that expression—"2".

But `i.e.` is fine too
> the result of that expression, i.e. "2".

This is merely a matter of taste, you can ignore my remark if you wish.